def fa():
    raise Exception('Traceback Test')

def runa():
    fa()
